library/java/net/openid/appauth/AuthorizationException.java

```diff
@@ -95,22 +94,24 @@
     /**
      * The error type for OAuth specific errors on the authorization endpoint. This error type is
      * used when the server responds to an authorization request with an explicit OAuth error, as
-     * defined by
-     * <a href="https://tools.ietf.org/html/rfc6749#section-4.1.2.1">The OAuth2 specification</a>.
-     * If the authorization response is invalid and not explicitly an error response, another error
-     * type will be used.
+     * defined by [the OAuth2 specification, section 4.1.2.1](
```

Comment:
Yeah - if you put the open parenthesis "(" on the next line, it doesn't work. However as long as this is on the same line, the tool seems to work.